src/components/FilmDetail.vue

```diff
@@ -1,39 +1,45 @@
 <template>
     <div>
-      <section>
-        <h2>Detail goes here</h2>
-        <div v-if="asyncDataStatusReady" class="row">
-          <div class="col-sm-6 col-md-4 col-xs-12">
-            <img 
-              :src="this.$store.state.filmInfo.poster_path | pathImage"
-              :alt="this.$store.state.filmInfo.title"
-              class="img-fluid"
-            >
+      <div class="search">
+        <Autocomplete />
+      </div>
+      <div>
+        <section>
+          <h2>Detail goes here</h2>
+          <div v-if="asyncDataStatusReady" class="row">
+            <div class="col-sm-6 col-md-4 col-xs-12">
+              <img 
+                :src="this.$store.state.filmInfo.poster_path | pathImage"
+                :alt="this.$store.state.filmInfo.title"
+                class="img-fluid"
+              >
+            </div>
+            <div class="col-sm-6 col-md-4 col-xs-12">
+              <h2>{{this.$store.state.filmInfo.original_title}}</h2>
+              <h1>Overview: </h1>
+              <p>{{this.$store.state.filmInfo.overview | trimmedString}}</p>
```

Comment:
Chỗ này nên viết ntn cho gọn anh này 
computed: {
  localComputed () { /* ... */ },
  // mix this into the outer object with the object spread operator
  ...mapState({
    // ...
  })
}
#

Reply:
ok e